README.md

```diff
@@ -22,14 +22,17 @@ The `webApp` directory contains a web application offering a small demo as well
 
 Palmetto can be used as a docker container.
 
-[The index](https://hobbitdata.informatik.uni-leipzig.de/homes/mroeder/palmetto/Wikipedia_bd.zip) should be downloaded and extracted to some path (for example, `/path/to/indexes`). First, the project has to be compiled. Enter the `webApp` directory and run:
+[The index](https://hobbitdata.informatik.uni-leipzig.de/homes/mroeder/palmetto/Wikipedia_bd.zip) should be downloaded and extracted to some path (for example, `/path/to/indexes`). 
+After that, the container can be run the following way:
 ```
-mvn clean package
+docker run -p 7777:8080 -d -v /path/to/indexes/:/usr/src/indexes/:ro dicegroup/palmetto-service
```

Comment:
Shouldn't the path contain `/local/` instead `/src/` like you wrote in https://github.com/dice-group/Palmetto/issues/49#issuecomment-885805722 ?

Reply:
You are right :open_mouth: 
Sorry, it seems like I created this PR in a hurry and overlooked this :disappointed: 

Thanks for having a look at it! :+1: 
It should be fixed with #55